package.json

```diff
@@ -13,6 +13,7 @@
 	},
 	"scripts": {
 		"setup": "composer install && yarn install && yarn run build && wp-scripts env install",
+		"start:dev": "NODE_ENV=development yarn start",
```

Comment:
This is not needed as the package we use - [`@wordpress/scripts`](https://github.com/WordPress/gutenberg/tree/wp/5.4/packages/scripts) - which defaults to `NODE_ENV=development` when `yarn start` is used and `NODE_ENV=production` when `yarn build`.

We need to upgrade this setup to the newest version in the future but the purpose is to stick to the core Gutenberg project standards as much as we can.